src/Core.Networking/NetworkingConstants.cs

```diff
@@ -2,5 +2,7 @@
 
 public static class NetworkingConstants
 {
-    public static int BufferSize = 4096;
+    public static readonly int BufferSize = 4096;
+    
+    public static readonly int PingIntervalInSeconds = 5;
```

Comment:
Is 5 seconds too long? I'm not sure about the duration.

Reply:
Should be fine. We can make it configurable in the future.